### PR TITLE
Past Reminder Fix

### PR DESCRIPTION
**Pull request by eren8204**
Fix for #991

### CODE REVIEW

File: omniNotes/src/main/java/it/feio/android/omninotes/utils/date/ReminderPickers.java [hunk 1]
```diff
@@ -62,11 +66,20 @@ public void onDateTimeRecurrenceSet(SelectedDate selectedDate, int hourOfDay, in
...
+        if (reminderTimeInMillis > currentTime) {
+
+          mOnReminderPickedListener.onReminderPicked(reminder.getTimeInMillis());
+          mOnReminderPickedListener.onRecurrenceReminderPicked(
+                  RecurrenceHelper
+                          .buildRecurrenceRuleByRecurrenceOptionAndRule(recurrenceOption, recurrenceRule));
+        }
+        else {
+          View rootView = mActivity.findViewById(android.R.id.content);
+          Snackbar.make(rootView, "Cannot set alarm in the past", Snackbar.LENGTH_LONG).show();
```
Comment (federicoiosue) [inline omniNotes/src/main/java/it/feio/android/omninotes/utils/date/ReminderPickers.java]: Please move this in resource strings to allow the message's translation

File: omniNotes/src/main/java/it/feio/android/omninotes/utils/date/ReminderPickers.java [hunk 2]
```diff
@@ -62,11 +66,20 @@ public void onDateTimeRecurrenceSet(SelectedDate selectedDate, int hourOfDay, in
         Calendar reminder = selectedDate.getFirstDate();
         reminder.set(Calendar.HOUR_OF_DAY, hourOfDay);
         reminder.set(Calendar.MINUTE, minute);
-
-        mOnReminderPickedListener.onReminderPicked(reminder.getTimeInMillis());
-        mOnReminderPickedListener.onRecurrenceReminderPicked(
-            RecurrenceHelper
-                .buildRecurrenceRuleByRecurrenceOptionAndRule(recurrenceOption, recurrenceRule));
+        long reminderTimeInMillis = reminder.getTimeInMillis();
```
Comment (federicoiosue) [inline omniNotes/src/main/java/it/feio/android/omninotes/utils/date/ReminderPickers.java]: No need to have these two variables in Java heap if they're used just here. Why not to change the whole block to something like:
```java
...
  if (reminder.getTimeInMillis() < System.currentTimeMillis()) {
    Snackbar.make(mActivity.findViewById(android.R.id.content), "Cannot set alarm in the past", Snackbar.LENGTH_LONG).show();
    return;
  }

  mOnReminderPickedListener.onReminderPicked(reminder.getTimeInMillis());
  mOnReminderPickedListener.onRecurrenceReminderPicked(
          RecurrenceHelper.buildRecurrenceRuleByRecurrenceOptionAndRule(recurrenceOption, recurrenceRule));
}
```
Less code, easier to read.

**Review by federicoiosue — CHANGES_REQUESTED** [review]
Great job, let's improve it a bit!

**Comment by eren8204** [comment]
@federicoiosue i've made the required cahnges, you can check now

**Pull request merged**